web/app/components/doc/inline.hbs

```diff
@@ -18,8 +19,10 @@
 
     <Person @ignoreUnknown={{true}} @imgURL={{@avatar}} @email={{@owner}} />
 
-    {{#if (and @isResult @snippet)}}
```

Comment:
Need to investigate/report this. Glint thought `@snippet` was undefined despite it passing the `and` check.